src/processor/operator/aggregate/base_aggregate_scan.cpp

```diff
@@ -23,5 +23,7 @@ void BaseAggregateScan::writeAggregateResultToVector(ValueVector& vector, uint64
     }
 }
 
+void BaseAggregateScan::finalize(ExecutionContext*) {}
```

Comment:
This should be unnecessary if you change `PhysicalOperator::finalize` to handle source operators.

Reply:
reverted